setup.py

```diff
@@ -140,34 +140,18 @@ def mark_lhotse_version(version: str) -> None:
 ]
 
 try:
-    # A matrix of compatible torch and torchaudio versions.
-    # If the user already installed torch, we'll try to find the compatible
-    # torchaudio version. If they haven't installed torch, we'll just install
-    # the latest torch and torchaudio.
-    # This code is partially borrowed from ESPnet's setup.py.
+    # If the user already installed PyTorch, make sure he has torchaudio too.
+    # Otherwise, we'll just install the latest versions from PyPI for the user.
     import torch
 
-    torch_ver = LooseVersion(torch.__version__)
-    if torch_ver >= LooseVersion("1.10.1"):
-        raise NotImplementedError("PyTorch version >= 1.10.1 is not yet supported")
-    elif torch_ver >= LooseVersion("1.10.0"):
-        install_requires.append("torchaudio==0.10.0")
-    elif torch_ver >= LooseVersion("1.9.1"):
-        install_requires.append("torchaudio==0.9.1")
-    elif torch_ver >= LooseVersion("1.9.0"):
-        install_requires.append("torchaudio==0.9.0")
-    elif torch_ver >= LooseVersion("1.8.2"):
-        install_requires.append("torchaudio==0.8.2")
-    elif torch_ver >= LooseVersion("1.8.1"):
-        install_requires.append("torchaudio==0.8.1")
-    elif torch_ver >= LooseVersion("1.8.0"):
-        install_requires.append("torchaudio==0.8.0")
-    elif torch_ver >= LooseVersion("1.7.1"):
-        install_requires.append("torchaudio==0.7.2")
-    else:
+    try:
+        import torchaudio
+    except ImportError:
         raise ValueError(
-            f"Lhotse requires torch>=1.7.1 and torchaudio>=0.7.2 -- "
-            f"please update your torch (detected version: {torch_ver})."
+            "We detected that you have already installed PyTorch, but haven't installed torchaudio. "
+            "Unfortunately we can't detect the compatible torchaudio version for you; "
+            "you will have to install it manually. "
+            "For instructions, please refer to https://pytorch.org/get-started/locally/"
```

Comment:
I would recommend putting the link about how to install torchaudio here, i.e.,
https://github.com/pytorch/audio#dependencies